tbconnect/tasks.py

```diff
@@ -17,20 +17,16 @@ def perform_sync_to_rapidpro():
         rapidpro = TembaClient(settings.RAPIDPRO_URL, settings.RAPIDPRO_TOKEN)
 
         # using data__contains to hit the GIN index - userprofile__data__gin_idx
-        for contact in (
-            HealthCheckUserProfile.objects.filter(
-                data__contains={"follow_up_optin": True}
-            )
-            .exclude(data__contains={"synced_to_tb_rapidpro": True})
-            .iterator()
-        ):
+        for contact in HealthCheckUserProfile.objects.exclude(
+            data__contains={"synced_to_tb_rapidpro": True}
+        ).iterator():
```

Comment:
Fixed this. It will now only include where synced_to_tb_rapidpro=False and that only gets set to False if we want to sync it here:
https://github.com/praekeltfoundation/healthcheck/pull/24/files#diff-a88a17ee2a7d158443fb2a2f016bf7dcR227